cli/lib/tasks/install.js

```diff
@@ -14,6 +14,7 @@ const unzip = require('./unzip')
 const logger = require('../logger')
 const la = require('lazy-ass')
 const is = require('check-more-types')
+// const cachedir = require('cachedir')
```

Comment:
we usually do not leave commented out code, because it is 
- just confusing
- not tested